.github/workflows/windows_dpcpp.yml

```diff
@@ -0,0 +1,65 @@
+name: Windows (Intel DPC++ Compiler, Python 3.11, Release)
+on:
+  workflow_dispatch:
+  pull_request:
+  merge_group:
+concurrency:
+  # github.ref is not unique in post-commit
+  group: ${{ github.event_name == 'push' && github.run_id || github.ref }}-windows-dpcpp
+  cancel-in-progress: true
+
+env:
+  PIP_CACHE_PATH: /mount/caches/pip/win
+  PYTHON_VERSION: '3.11'
+
+permissions: read-all
+
+jobs:
+  Smart_CI:
+    runs-on: ubuntu-latest
+    outputs:
+      affected_components: "${{ steps.smart_ci.outputs.affected_components }}"
+      skip_workflow: "${{ steps.smart_ci.outputs.skip_workflow }}"
+    steps:
+      - name: checkout action
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+        with:
+          sparse-checkout: .github/actions/smart-ci
+
+      - name: Get affected components
+        id: smart_ci
+        uses: ./.github/actions/smart-ci
+        with:
+          repository: ${{ github.repository }}
+          pr: ${{ github.event.number }}
+          commit_sha: ${{ github.sha }}
+          ref_name: ${{ github.ref_name }}
+          component_pattern: "category: (.*)"
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          skip_when_only_listed_labels_set: 'docs'
+          skip_when_only_listed_files_changed: '*.md,*.rst,*.png,*.jpg,*.svg,*/layer_tests_summary/*,*/conformance/*'
+
+  Build:
+    needs: [ Smart_CI ]
+    if: "!needs.smart_ci.outputs.skip_workflow"
+    uses: ./.github/workflows/job_build_windows.yml
+    with:
+      runner: 'aks-win-16-cores-32gb'
+      affected-components: ${{ needs.smart_ci.outputs.affected_components }}
+      build-type: 'Release'
+      compiler-type: 'DPCPP'
+
+
+  Overall_Status:
+    name: ci/gha_overall_status_windows
```

Comment:
It should have a different name from that of in `windows.yml` because the required checks are set up using these names.